examples/EU DST example

```diff
@@ -0,0 +1,52 @@
+/* EU DST example */
```

Comment:
This deserves a few more explanations, stating for instance that:

1. The RTC is kept in local winter time. This is not obvious at first sight, and is quite non-standard (common practice is to keep the RTC either in UTC or in local time).

2. This code is for CET only. It can work across Europe if one is willing to accept the DST change to be one hour off.

Reply:
Very relevant suggestions. 
Point 1 appears to be the key to make it a very simple procedure. 
Point 2 The hour is now set to 1 and 3. This can be adapted to your local time (for instance at 2 and 4) if applicable. Therefore it is an example. 
I would love to add these point to a comment in the proposed code, but have problems to find out how. (This is the very first pull request I ever do).

Reply:
> This can be adapted to your local time

I believe a comment explaining how to do so would be of some value.